vllm/engine/async_llm_engine.py

```diff
@@ -806,7 +806,7 @@ async def generate(
             request_id: The unique id of the request.
             lora_request: LoRA request to use for generation, if any.
             trace_headers: OpenTelemetry trace headers.
-            prompt_adapter_request: Prompt Adapter request to use 
+            prompt_adapter_request: Prompt Adapter request to use
                                             for generation, if any.
```

Comment:
```suggestion
            prompt_adapter_request: Prompt Adapter request to use
                for generation, if any.
```

Reply:
hmm, idk why ruff format this 🤔

Reply:
Which rule is this under?

Reply:
This is their rules afaict. There aren't ways to ignore this other than `# ruff: format: ignore`

Reply:
Ok let's just follow their rules then. No big deal as long as the docstring still appears correctly.